docs/book/src/cronjob-tutorial/testdata/project/config/crd/bases/batch.tutorial.kubebuilder.io_cronjobs.yaml

```diff
@@ -15,6549 +15,7177 @@ spec:
+                                                spec named by the GMSACredentialSpecName
+                                                field.
+                                              type: string
+                                            gmsaCredentialSpecName:
+                                              description: GMSACredentialSpecName
+                                                is the name of the GMSA credential
+                                                spec to use.
+                                              type: string
+                                            runAsUserName:
+                                              description: The UserName in Windows
+                                                to run the entrypoint of the container
+                                                process. Defaults to the user specified
+                                                in image metadata if unspecified.
+                                                May also be set in PodSecurityContext.
+                                                If set in both SecurityContext and
+                                                PodSecurityContext, the value specified
+                                                in SecurityContext takes precedence.
+                                              type: string
+                                          type: object
+                                      type: object
+                                    startupProbe:
+                                      description: 'StartupProbe indicates that the
+                                        Pod has successfully initialized. If specified,
+                                        no other probes are executed until this completes
+                                        successfully. If this probe fails, the Pod
+                                        will be restarted, just as if the livenessProbe
+                                        failed. This can be used to provide different
+                                        probe parameters at the beginning of a Pod''s
+                                        lifecycle, when it might take a long time
+                                        to load data or warm a cache, than during
+                                        steady-state operation. This cannot be updated.
+                                        This is a beta feature enabled by the StartupProbe
+                                        feature flag. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes'
+                                      properties:
+                                        exec:
+                                          description: One and only one of the following
+                                            should be specified. Exec specifies the
+                                            action to take.
+            
```

Comment:
Just have to confirm, this is meant to be over 7k lines for this CRD?

Reply:
Since the diff is so big it's hard to load in PR reviewing - https://github.com/kubernetes-sigs/kubebuilder/blob/8bd0cbfd38d7027b1bf60d3f156f0d4f00f654db/docs/book/src/cronjob-tutorial/testdata/project/config/crd/bases/batch.tutorial.kubebuilder.io_cronjobs.yaml for clarity since this comment shows that it's on line 1923.

Reply:
It is generated by the command. ( make manifests ) So, I am not worried about it.

Reply:
The crd is generated via make manifest. Shows that it has been getting the doc/texts to be generated.

Reply:
The CRD are generating via make manifests so it shows fine. 
To ensure that I re-generated the project locally with the helpers ([generate_multiversion.sh](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/generate_multiversion.sh) and [generate_cronjob.sh](https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/cronjob-tutorial/testdata/generate_cronjob.sh)) scripts and executed make manifests before add the docs comments.